src/big_mess/loaders.py

```diff
@@ -0,0 +1,62 @@
+from typing import Optional
+from pyspark.sql import SparkSession, DataFrame as SparkDataFrame, functions as F
+from pyspark.sql.types import IntegerType, FloatType, StringType, StructType
+
+
+def default_loader(spark: SparkSession, file_path: Optional[str] = None ) -> SparkDataFrame:
+    
+    columns = ['lon', 'lat', 'Date', 'Rainf', 'Evap', 'AvgSurfT', 'Albedo','SoilT_10_40cm', 'GVEG', 'PotEvap', 'RootMoist', 'SoilM_100_200cm',
+                #"SoilT_40_100cm", #FIXME uncomment afete clickup task 86bwwntfk - "Przywrócić kolumnę 'SoilT_40_100cm' do datasetu"
```

Comment:
Zmienić "SoilT_10_40cm" na "SoilT_40_100cm".

Reply:
Na dysku zostały również wygenerowane 4 csv z błędnie nazwaną kolumną z powodu nieprawidłowego szablonu, być  może również tam warto poprawić nazwę kolumny.

Reply:
Otóż nie. Ani w researchu Mariusza, ani w Twojej selekcji cech nie było mowy o SoilT_10_40cm. Mariusz zaproponował SoilT_40_100cm LUB SoilT_100_200cm, Ty w swojej selekcji cech zasugerowałaś SoilT_40_100cm, więc taką też zostawiłam, ale przy tworzeniu nowego szablonu danych po prostu źle ją nazwałam. Mój błąd.

Reply:
Aha, ja się zasugerowałam nazwą tasku "Przywrócenie kolumny x" 😉  i dlatego myślałam, że mamy nie tę kolumnę co trzeba, a nie tylko niewłaściwą nazwę kolumny 😉

Reply:
Wniosek jest taki, że należy zamienić "SoilT_10_40cm" na "SoilT_40_100cm", chodzi tylko o zmianę nazwy... tak jak to napisałam w pierwszym komentarzu :)